feat(social-media): Add profiles use-case

jnv opened the pull request:
<!--- Tip: You don't have to remove these comments -->

## Description

<!--- Describe your changes in detail -->

## Motivation and Context

<!--- Why is this change required? What problem does it solve? -->
<!--- If it fixes an open issue, please link to the issue here. -->

## Types of changes

<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
<!--- If your changes are only to the internals then make sure any function has its attached documentation updated or added (e.g. descriptive name, doc comment, etc.). If your changes are to user-facing APIs or concepts then make sure README.md is updated accordingly (e.g. command help output). -->

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I haven't repeated the code. (DRY)
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

janhalama commented on grid/social-media/profiles/maps/twitter.suma:
```diff
@@ -0,0 +1,132 @@
+    response 200 "application/json" {
+      profiles = body.data.map((user) => {
+        const metrics = user.public_metrics || {};
+        return {
```
I miss mapping of `name` property (Display name of the profile).

jnv replied:
Good catch, thanks!

janhalama commented on grid/social-media/profiles/maps/twitter.test.ts:
```diff
@@ -0,0 +1,12 @@
+import { profilesTest } from './profiles';
+
+const profileIds = [
```
I like profile examples you have chosen 👍

janhalama reviewed (approved):
Looks good, I found just one minor issue. 👍

jnv commented:
@janhalama Thanks for the review! I have fixed the name mapping and also added a `location` field to avoid the need for minor version change in the future. I'll merge it now.
